source/index.ts

```diff
@@ -19,7 +19,7 @@ export interface OutputOptions {
   allow?: string[];
 }
 
-const packageRegex = /^\s*│\s*Package\s*│\s*(\S+)\s*│\s*$/gm;
+const packageRegex = /^\s*│?\s*Package\s*│?\s*(\S+)\s*│?\s*$/gm;
```

Comment:
Perhaps clamping the whitespace checking would be a good idea as part of this, it looks like the npm cli passes forward a unicode flag which determines pipe vs a blank space, hence `(?:\│|\s)`
```suggestion
const heading = 'Package';
const colWidths = [15, 62];
const maxIndent = 16;
const col1Pad = colWidths[0] - heading.length;
const packageRegex = new RegExp(
  `^\\s{0,${maxIndent}}(?:\\│|\\s)\\s{1,${col1Pad}}${heading}\\s{1,${col1Pad}}(?:\\│|\\s)\\s{1,${colWidths[1]}}(\\S{1,${colWidths[1]}})`,
  'gm'
);
``` 
Ranges are based on:
- https://github.com/yarnpkg/yarn/blob/ac21dbf16a5d2d5f4cb032f5da5313949db3fb71/src/reporters/console/console-reporter.js#L32
- https://github.com/npm/npm-audit-report/blob/139002428f52ca5c08cbbe71e68fd31693672907/reporters/detail.js#L107
- https://github.com/cli-table/cli-table3/blob/master/src/utils.js#L225-L226
- `maxIndent` is just an arbitrary upper limit.

Reply:
Nice! I didn't think to dig into the audit code for each package managers - some good finds in there! Knowing the tables render using the same boundaries, clamping down the white space is definitely the way to go 👍